app/controllers/sessions_controller.rb

```diff
@@ -39,10 +39,12 @@ def redirect_if_logged_in
   end
 
   def handle_success user
+    forwarding_url = session[:forwarding_url]
     reset_session
     log_in user
+    params[:session][:remember_me] == "1" ? remember(user) : forget(user)
```

Comment:
```suggestion
    params.dig(:session, :remember_me) == REMEMBER_ME ? remember(user) : forget(user)
```
E đưa string 1 này ra biến const nha. Dùng dig để truy cập deep hash